backends/advanced/src/advanced_omi_backend/routers/modules/annotation_routes.py

```
@@ -0,0 +1,318 @@
+        # Update memory content if accepted
+        if annotation.status == AnnotationStatus.ACCEPTED:
+            try:
+                await memory_service.update_memory(
+                    memory_id=annotation_data.memory_id,
+                    content=annotation_data.corrected_text,
+                    user_id=current_user.user_id,
+                )
+                logger.info(
+                    f"Updated memory {annotation_data.memory_id} with corrected text"
+                )
+            except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)